overlord/ifacestate/ifacestate_test.go

```diff
@@ -471,9 +471,10 @@ func (s *interfaceManagerSuite) TestBatchConnectTasks(c *C) {
 	connOpts := make(map[string]*ifacestate.ConnectOpts)
 
 	// no connections
-	ts, err := ifacestate.BatchConnectTasks(s.state, snapsup, conns, connOpts)
+	ts, hasHooks, err := ifacestate.BatchConnectTasks(s.state, snapsup, conns, connOpts)
```

Comment:
minor nitpick, you could call this `hasInterfaceHook`, but feel free to do that in a followup if you prefer since this is green

Reply:
Thanks, will do in a followup.